src/modules/endpoint/DatabaseConnector.js

```diff
@@ -381,15 +393,28 @@ class DatabaseConnector {
     const json = JSON.parse(await this.readFile(path))
 
     return json.map((jsonObject: ContentPoiJsonType) => {
+      const jsonLocation = jsonObject.location
+      // $FlowFixMe https://github.com/facebook/flow/issues/5838
+      const availableLanguages = new Map<string, string>(Object.entries(jsonObject.availableLanguages))
       return new PoiModel({
         path: jsonObject.path,
         title: jsonObject.title,
         content: jsonObject.content,
         thumbnail: jsonObject.thumbnail,
-        availableLanguages: jsonObject.availableLanguages,
+        availableLanguages,
         excerpt: jsonObject.excerpt,
-        location: jsonObject.location,
-        lastUpdate: moment(jsonObject.last_update, moment.ISO_8601),
+        location: new LocationModel({
+          name: null, // todo: NATIVE-549
```

Comment:
@klinzo should we disable the fetching of events until 549 is done? Maybe just comment out the fetching/persisting code?

Reply:
Do you mean disabling fetching of pois? @TechPiranja is fetching of pois initiated anywhere right now with this PR?

Reply:
@klinzo  no, the pois are not being fetched currently.

It **would have been** fetched if i added the "FETCH_POI" action type inside the cityContentReducer.
( which is currently not the case, but i stashed that code for the Poi UI Ticket )

Reply:
@maxammann so I guess we can merge this, right?